modules/op-release-notes-1-9.adoc

```diff
@@ -433,4 +433,18 @@ With this update, {pipelines-title} General Availability (GA) 1.9.2 is available
 [id="fixed-issues-1-9-2_{context}"]
 == Fixed issues
 
-* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
\ No newline at end of file
+* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
+
+[id="release-notes-1-9-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.3
+
+With this update, {pipelines-title} General Availability (GA) 1.9.3 is available on {product-title} 4.10, 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-9-3_{context}"]
+== Fixed issues
+
+* This update fixes the performance issues for more complex pipelines. Now, the CPU usage is reduced by 61% and the memory usage is reduced by 44%.
```

Comment:
These gains were observed for huge pipelines only.

Reply:
@khrm I wanted to avoid the word "huge" by adding "more complex" instead. But if that is not accurate enough, I have changed it to "huge" as suggested. Thank you!